polkadot/runtime/westend/src/lib.rs

```diff
@@ -1696,6 +1749,7 @@ mod benches {
 		[runtime_parachains::initializer, Initializer]
```

Comment:
You'll need `[runtime_parachains::coretime, Coretime]` here and the corresponding file in weights to generate the weights